service/src/main/java/bio/terra/workspace/app/controller/WorkspaceApiController.java

```diff
@@ -758,7 +758,7 @@ public ResponseEntity<ApiWsmPolicyMergeCheckResult> mergeCheck(
               targetWorkspaceId, CloudPlatform.fromApiCloudPlatform(platform));
 
       for (var existingResource : existingResources) {
-        if (!validRegions.contains(existingResource.getRegion())) {
+        if (!validRegions.stream().anyMatch(existingResource.getRegion()::equalsIgnoreCase)) {
```

Comment:
Seems a waste of a hashset to iterate through it, but I guess we have mixed case valid regions?